quick-install/copy-certs.sh

```diff
@@ -3,16 +3,22 @@ set -eo pipefail
 unset CDPATH
 IFS=$'\n\t'
 
+namespace='kube-public'
+if [ ! -z "$1" ]
+then
+	namespace=$1
+fi
+
 echo
 echo "Copying certs to nodes"
 
 #delete any old jobs
-for job in $(kubectl get jobs -n kube-public -o go-template --template '{{range .items}}{{.metadata.name}}
+for job in $(kubectl get jobs -n $namespace -o go-template --template '{{range .items}}{{.metadata.name}}
```

Comment:
I *think* we should put `$namespace` in double quotes. I guess there is as a potential security hole if an attacker can set the namespace to something like `bla; kubectl delete...`, but I don't think it's worth worrying about that one.

Reply:
I think we can assume that cluster admins will install trow in the cluster. Then this issue is degraded from security issue to nice-to-have :) Nevertheless, I will add double quotes, that's a good practice indeed.